phpunit/test-cdn-assets.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+class Test_CDN_Assets extends WP_UnitTestCase {
+
```

Comment:
This needs a setUp/tearDown method to check/reset the environmental WP_ENVIRONMENT_TYPE value.